app/javascript/packages/polyfill/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @typedef Polyfill
+ *
+ * @prop {()=>boolean} test Test function, returning true if feature is detected as supported.
+ * @prop {()=>Promise} load Function to load polyfill module.
+ */
+
+/**
+ * @typedef {"fetch"} SupportedPolyfills
+ */
+
+/**
+ * @type {Record<SupportedPolyfills,Polyfill>}
+ */
+const POLYFILLS = {
+  fetch: {
+    test: () => 'fetch' in window,
+    load: () => import(/* webpackChunkName: "whatwg-fetch" */ 'whatwg-fetch'),
+  },
+};
+
+/**
+ * Given an array of supported polyfill names, loads polyfill if necessary. Returns a promise which
+ * resolves once all have been loaded.
+ *
+ * @param {SupportedPolyfills[]} polyfills Names of polyfills to load, if necessary.
+ *
+ * @return {Promise}
+ */
+export function loadPolyfills(polyfills) {
+  return Promise.all(
```

Comment:
So I'm assuming we're not worrying about Polyfilling the Promise API 😂

Reply:
Just saw the note about core-js handling this for us --- makes more sense now https://github.com/18F/identity-idp/pull/4079/files#r471547471

Reply:
Yep, `Promise` should be provided by how Babel handles `useBuiltIns`. 👍